lib-multisrc/wpcomics/src/eu/kanade/tachiyomi/multisrc/wpcomics/WPComics.kt

```diff
@@ -171,6 +173,15 @@ abstract class WPComics(
                 val calendar = Calendar.getInstance()
 
                 when {
+                    yearWords.doesInclude(trimmedDate[1]) -> calendar.apply {
+                        add(Calendar.YEAR, -trimmedDate[0].toInt())
+                        set(Calendar.DAY_OF_MONTH, 1)
+                        set(Calendar.MONTH, Calendar.JANUARY)
+                    }
+                    monthWords.doesInclude(trimmedDate[1]) -> calendar.apply {
+                        add(Calendar.MONTH, -trimmedDate[0].toInt())
+                        set(Calendar.DAY_OF_MONTH, 1)
+                    }
```

Comment:
you can simply subtract the year or month respectively, no need to also change day (and month)